install/Knative-with-Minishift.md

```diff
@@ -79,9 +88,9 @@ minishift docker-env
 minishift oc-env
 ```
 
-## Preparing Knative Deployment
+### Preparing Knative Deployment
```

Comment:
Should not this be same level as `Automatic Set Up`? As I can see both `Automatic Set Up` and `Manually Set Up` need to `Preparing Knative Deployment`.

Reply:
We don't need this heading at all. (Best practice is to avoid two headings on top of each other.) I recommend deleting the heading on line 91.